<launch>
  <node name="hk_camera_node" pkg="hk_camera" type="hk_camera_node" output="screen" >
    <!-- <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="camera_frame_id" value="usb_cam" /> -->
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/hk_camera/image_raw"/>
    <param name="autosize" value="true" />
  </node>
</launch>
